gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -159,6 +159,130 @@
+
+            <p class="text-info" ng-if="webauthn.errors.aborted" translate>Registration aborted by user.</p>
+
+            <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>Registration failed - possibly aborted, timed out, or blocked by the browser.</p>
+
+            <div ng-if="webauthn.errors.securityError">
+              <p class="text-danger" ng-if="webauthn.errors.securityError" translate>Initialization blocked by browser. Please make sure Syncthing is correctly configured.</p>
+              <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>Syncthing Documentation: Customizing the GUI address</a>
+            </div>
+
+            <p class="text-danger" ng-if="webauthn.errors.registrationFailed" translate>Failed to create credential, see Syncthing logs for details.</p>
+            <p class="text-danger" ng-if="webauthn.errors.registrationTimedOut" translate>Registration timed out. Please try again.</p>
+
+            <p class="text-info" ng-if="webauthnAvailable() && isRawIpAddress()" translate>WebAuthn is not supported on raw IP address hosts.</p>
+
+            <p class="text-info" ng-if="!isLocationInsecure() && !webauthnAvailable()" translate>Your browser does not support WebAuthn.</p>
+
+            <p class="text-info" ng-if="!locationMatchesWebauthnOrigin()" translate translate-value-address="{{ getWebauthnOrigin() }}">Reload page at {%address%} to enable WebAuthn.</p>
+
+            <p ng-if="isLocationInsecure() || !locationMatchesWebauthnOrigin()">
+              <button ng-if="tmpGUI && settingsModified()" type="button" class="btn btn-primary" ng-click="reloadSettingsAtWebauthnAddress(true)"><span class="fas fa-check"></span>&nbsp;<span translate translate-value-address="{{ getWebauthnOrigin() }}">Save settings and reload page at {%address%}</span>
+              </button>
+
+              <button ng-if="tmpGUI && !settingsModified()" type="button" class="btn btn-primary" ng-click="reloadSettingsAtWebauthnAddress(false)" translate translate-value-address="{{ getWebauthnOrigin() }}">Reload page at {%address%}</button>
+            </p>
+
+            <!-- Ineligible WebAuthn credentials -->
+            <div ng-repeat="_ in tmpGUI.webauthnCredentials | filter:{rpId: '!'+tmpGUI.webauthnRpId}:true | limitTo:1">
+              <label translate>Ineligible WebAuthn Credentials (Passkeys)</label>
+              <p translate>The credentials below were created with a different setting of "Webauthn Rp Id" in advanced settings. They cannot be used without restoring this setting to the value shown as "RP ID" below. Note that doing so would make any credentials above ineligible instead.</p>
+              <div class="webauthn-credentials two-column-flexbox">
+                <div class="webauthn-credential-card panel panel-default" ng-repeat="cred in tmpGUI.webauthnCredentials | filter:{rpId: '!'+tmpGUI.webauthnRpId}:true">
```

Comment:
Am I missing something, or are these two `webauthn-credential`... classes never defined?

I was starting to investigate because of a thin grey line just below the "Add passkey" link, which seems to be a transparent(!) bottom border around the next `webauthn-credential-card` which contains the link.

Reply:
They were defined earlier, but it looks like they were left behind in de0abe98bef9ba100257921aecc1987c56f81bc7. Removed in 531b448697e3963652537f37cd0907fe05135148, thanks!